hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/props/NodeConnectInfo.java

```diff
@@ -29,6 +29,7 @@ public class NodeConnectInfo {
     public static int NEXT_DEFAULT_ACCOUNT_NUM = 3;
     private static final int DEFAULT_PORT = 50211;
     private static final int DEFAULT_TLS_PORT = 50212;
+    private static final int DEFAULT_NODE_OPERATOR_PORT = 50213;
```

Comment:
I would suggest removing changes to this class; although not marked `@Deprecated`, it really should be eliminated in favor of getting node connection info directly from a `HederaNode` object.

Reply:
Reverted the changes and added `@Deprecated` with comment to use `HederaNode` object for  getting node connection info.